build.gradle

```diff
@@ -7,41 +12,175 @@ apply plugin: 'idea'
 apply plugin: 'maven-publish'
 
 def DERBY_VERSION = '10.14.1.0'
+def DERBY_MINOR = '10.15'
```

Comment:
I'd suggest also pinning a specific revision, this way we can tell what code was used from the git sha.

Reply:
For revision do you mean "svn revision" ?